uber/configspec.ini

```diff
@@ -509,14 +509,9 @@ page_viewed = string(default='pageview')
 [[food_restriction]]
```

Comment:
I moved all of this out of ``configspec.ini`` and only left the options which have specific logic (which we'd expect any event to care about).  These options all exist in our ``production-config`` plus some new ones.